index.js

```diff
@@ -239,7 +239,12 @@ class K8sVMExecutor extends Executor {
             LOW: this.lowCpu,
             MICRO: this.microCpu
         };
-        const CPU = (cpuConfig in cpuValues) ? cpuValues[cpuConfig] : cpuValues.LOW;
+        let cpu = (cpuConfig in cpuValues) ? cpuValues[cpuConfig] : cpuValues.LOW;
+
+        // allow custom cpu value
+        if (Number.isInteger(cpuConfig)) {
+            cpu = cpuConfig;
```

Comment:
lets have an upper threshold of 32.

Reply:
set the default max as turbo value. we will override that in our internal worker config